spec.html

```diff
@@ -21782,11 +21782,11 @@ <h1>
       <emu-clause id="sec-forbodyevaluation" type="abstract operation">
         <h1>
           ForBodyEvaluation (
-            _test_: unknown,
-            _increment_: unknown,
-            _stmt_: unknown,
-            _perIterationBindings_: unknown,
-            _labelSet_: unknown,
+            _test_: an |Expression| Parse Node or ~[empty]~,
```

Comment:
I looked into this and found it really weird, so I opened https://github.com/tc39/ecma262/pull/2862.

Reply:
https://github.com/tc39/ecma262/pull/2862 has been merged now, so we'll need to update this to `~empty~`.